### PR TITLE
Make nameless unions really nameless

### PR DESCRIPTION
**Pull request by Joshua-Ashton**
This makes it consistent across tests and vkd3d.

### CODE REVIEW

File: build/vkd3d_version.c [hunk 1]
```diff
@@ -0,0 +1 @@
+const char vkd3d_build[] = "v1.1";
```
Comment (HansKristian-Work) [inline build/vkd3d_version.c:1]: Is this a leftover from Meson PR?

Comment (Joshua-Ashton) [reply build/vkd3d_version.c:1]: oops, appears to be!

Comment (HansKristian-Work) [reply build/vkd3d_version.c:1]: Could you also please add the "Signed-off-by:" line, to be consistent with other commits.

Comment (Joshua-Ashton) [reply build/vkd3d_version.c]: Done

**Pull request merged**